zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java

```diff
@@ -628,7 +628,7 @@ public void broadcastParagraph(Note note, Paragraph p) {
   }
 
   public void broadcastParagraphs(Map<String, Paragraph> userParagraphMap,
-      Paragraph defaultParagraph) {
+                                  Paragraph defaultParagraph) {
```

Comment:
@jongyoul I'v just removed the `defaultParagraph` parameter which is not used at all in b1e782e